tests/test_models/test_inference/test_forecast.py

```diff
@@ -423,19 +428,19 @@ def _test_forecast_out_sample_prefix(ts, model, transforms, full_prediction_size
 
         forecast_full_ts = ts.make_future(future_steps=full_prediction_size, tail_steps=model.context_size)
         if isinstance(model, get_args(ContextRequiredModelType)):
-            model.forecast(forecast_full_ts, prediction_size=full_prediction_size)
+            forecast_full_ts = model.forecast(forecast_full_ts, prediction_size=full_prediction_size)
         else:
-            model.forecast(forecast_full_ts)
+            forecast_full_ts = model.forecast(forecast_full_ts)
 
         # forecasting only prefix
         torch.manual_seed(11)  # TODO: remove after fix at issue-802
 
         forecast_prefix_ts = ts.make_future(future_steps=full_prediction_size, tail_steps=model.context_size)
         forecast_prefix_ts.df = forecast_prefix_ts.df.iloc[:-prediction_size_diff]
         if isinstance(model, get_args(ContextRequiredModelType)):
```

Comment:
This substring looks like we can define as a function and reuse it everywhere?

Reply:
Fixed.